modules/ROOT/pages/kafka/kafka-connector-reference-3.0.adoc

```diff
@@ -13,262 +13,325 @@ Release Notes: xref:release-notes::connector/kafka-connector-release-notes-mule-
-| Additional JAAS Properties a| Object |  Additional properties such as a key-value pair that you need to set in `sasl.jaas.config` and that you usually include in the JAAS configuration file. |  | 
+| Additional JAAS Properties a| Object |  Additional properties as key-value pairs, set in `sasl.jaas.config`, and that you usually include in the JAAS configuration file. |  | 
 |===
 
-[[kafka-consumer-config_kerberos-ssl-kafka-consumer-connection]]
+[[KafkaConsumerConfig_KerberosSslKafkaConsumerConnection]]
 ===== Kafka Kerberos SSL Consumer Connection
 
+
 ====== Parameters
-[%header,cols="20s,20a,35a,20a,5a"]
+
+[%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Consumer Partitions a| Number |  The number of partitions to use for the consumer. |  1 | 
+| Consumer Partitions a| Number |  The number of partitions to use for the consumer. |  `1` | 
 | Group Id a| String |  A unique string that identifies the consumer group this consumer belongs to. |  | x
 | Bootstrap Servers a| String |  Comma-separated host-port pairs used for establishing the initial connection to the Kafka cluster. The same as the `bootstrap.servers` value you must provide to Kafka clients (producer and consumer). |  | x
-| Additional Properties a| Object |  Additional properties such as a key-value pair that you need for your connection. |  | 
+| Additional Properties a| Object |  Additional properties, as key-value pairs, that you need for your connection. |  | 
 | Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
-| Key Store Type a| String |  The file format of the key store file. This is optional and the default value is `JKS`. |  `JKS` | 
-| Key Store Password a| String |  The store password for the key store file. This is optional and needed only if *Key Store Location* is configured. |  | 
-| Key Store Location a| String |  The location of the key store file. This is optional and can be used for two-way authentication for the connector. |  | 
-| Trust Store Type a| String |  The file format of the trust store file. |  `JKS` | 
-| Trust Store Password a| String |  The password for the trust store file. |  | x
-| Trust Store Location a| String |  The location of the trust store file. |  | x
-| Kerberos Config File a| String |  Kerberos configuration file. The essential Kerberos configuration information is the default realm and the default KDC. |  | 
-| Principal a| String |  Kerberos principal |  | x
-| Keytab a| String |  Path to keytab file associated with the `principal`. |  | x
+| Key Store Type a| String |  (Optional) The file format of the keystore file. | `JKS` | 
+| Key Store Password a| String |  The store password for the keystore file. This is optional and needed only if `keyStoreLocation` is configured. |  | 
```

Comment:
"stored"

Reply:
This is actually the password for the keystore